libraries/lib-datahandler/src/main/java/com/siemens/sw360/datahandler/common/SW360Utils.java

```diff
@@ -314,4 +317,51 @@ public boolean apply(String input) {
         }
         return idToLicense;
     }
+
+    public static String fieldValueAsString(Object fieldValue) {
+        if (fieldValue instanceof TEnum) {
+            return nullToEmpty(ThriftEnumUtils.enumToString((TEnum) fieldValue));
+        }
+        if (fieldValue instanceof String) {
+            return nullToEmpty((String) fieldValue);
+        }
+        if (fieldValue instanceof Map) {
+            List<String> mapEntriesAsStrings = nullToEmptyMap(((Map<String, Object>) fieldValue)).entrySet().stream()
+                    .map(e -> {
+                        String valueString = e.getValue() != null ? e.getValue().toString():"";
+                        return e.getKey() + " : " + valueString;
+                    })
+                    .collect(Collectors.toList());
+            return joinStrings(mapEntriesAsStrings);
+        }
+        if (fieldValue instanceof Iterable){
+            return joinStrings((Iterable<String>) fieldValue);
+        }
+        return fieldValue.toString();
+    }
+
+    public static String displayNameFor(String name, Map<String, String> nameToDisplayName){
+        return nameToDisplayName.containsKey(name)? nameToDisplayName.get(name) : name;
+    }
+
+    public static <T> Map<String, T> putReleaseNamesInMap(Map<String, T> map, List<Release> releases) {
+        if(map == null || releases == null) return Collections.emptyMap();
+        Map<String, T> releaseNamesMap = new HashMap<>();
+        releases.stream()
+                .forEach(r -> releaseNamesMap.put(printName(r),map.get(r.getId())));
```

Comment:
True, but this means again more clicking. It's a very short method and without collector you can see what it does. Can it stay as it is?